python/setup.py

```diff
@@ -87,7 +87,9 @@ def git_describe_version(original_version):
 
 
 LIB_LIST, __version__ = get_lib_path()
+print("%%%%%%%%%%%%%%%%%%%%%%%: ", __version__)
 __version__ = git_describe_version(__version__)
+print("#######################: ", __version__)
```

Comment:
Remove

Reply:
Hi @masahi , thank you for looking at this! I have removed this.